apps/avatax/src/modules/saleor/order.ts

```diff
@@ -47,3 +51,51 @@ export class SaleorOrder {
     return this.data.order.channel.taxConfiguration.pricesEnteredWithTax;
   }
 }
+export class SaleorCancelledOrder {
+  private static schema = z.object({
+    order: z.object({
+      channel: z.object({
+        id: z.string(),
+        slug: z.string(),
+      }),
+      id: z.string(),
+      avataxId: z.string(),
```

Comment:
**Question:** do Saleor sends avatax id in payload?

Reply:
AFAIR its mapped metadata key

Reply:
For `order-cancel` yes. It's missing when for example order has not been calculated with Avatax